app/editor/src/features/content/form/constants/showOnlyOptions.ts

```diff
@@ -0,0 +1,11 @@
+import { OptionItem } from 'tno-core';
+
+import { ShowOnlyValues } from './ShowOnlyValues';
+
+export const showOnlyOptions = [
+  new OptionItem('Print Content', ShowOnlyValues.PrintContent),
+  new OptionItem('Included in EoD', ShowOnlyValues.IncludedInEod),
+  new OptionItem('On Ticker', ShowOnlyValues.PrintContent),
```

Comment:
Don't think this makes sense.  I believe also they wanted to remove 'On Ticker' anyway.

Reply:
Woops, that was a copy paste error! Good catch - will remove on ticker from both checkboxes and dropdown while I am there though!